builtin/providers/aws/resource_aws_vpn_gateway.go

```diff
@@ -0,0 +1,328 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/aws-sdk-go/aws"
+	"github.com/hashicorp/aws-sdk-go/gen/ec2"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsVpnGateway() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsVpnGatewayCreate,
+		Read:   resourceAwsVpnGatewayRead,
+		Update: resourceAwsVpnGatewayUpdate,
+		Delete: resourceAwsVpnGatewayDelete,
+
+		Schema: map[string]*schema.Schema{
+			"availability_zone": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"type": &schema.Schema{
+				Type:     schema.TypeString,
+                Default:  "ipsec.1",
```

Comment:
According to the [VPN Gateway documentation][1],  `ipsec.1` is the only valid option for `type` here. Is there a good case for even offering this configuration field? 

[1]: http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateVpnGateway.html